server/cmwell-cons/src/main/scala/Grid.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.Try
 
 case class Grid(user: String,
                 password: String,
-                ipMappings: IpMappings,
+                clusterIps: Seq[String],
```

Comment:
let's leave it as a `Seq`, but add in case class CTOR enforcement for uniqueness, e.g. `require(ips.distinct sameElements ips, "must be uniqe")`.

Reply:
done